packages/web-shared/components/Searchbar/Search.styles.js

```diff
@@ -84,12 +84,14 @@ const Wrapper = withTheme(styled.div`
     border-radius: 0;
   }
 
-  .chip-list-feature {
-    * > &:first-child {
+  .chip-list-feature ul {
+    overflow-x: scroll;
+
+    & > *:first-child {
       margin-left: 1rem;
     }
-    * > &:last-child {
-      margin-left: 1rem;
+    & > *:last-child {
```

Comment:
I wrote this CSS in the previous PR, had it working, then moved it to a different section and butchered it when re-writing 😅